tests/01_03_06-2wp_no_locking_cap.js

```diff
@@ -1,116 +1,83 @@
-      throw new CustomError('Transfer BTC to RBTC failure', err);
-    }
+    const btcAddressInfo = await btcTxHelper.generateBtcAddress('legacy');
+    await whitelistingAssertions.assertAddLimitedLockWhitelistAddress(rskTxHelper, btcAddressInfo.address, PEGIN_VALUE_IN_SATOSHI);
+    await rskUtils.mineAndSync(rskTxHelpers);
+    
+    const recipientRskAddressInfo = getDerivedRSKAddressInformation(btcAddressInfo.privateKey, btcTxHelper.btcConfig.network);
+    await rskTxHelper.importAccount(recipientRskAddressInfo.privateKey);
+    const unlocked = await rskTxHelper.unlockAccount(recipientRskAddressInfo.address);
+    expect(unlocked, 'Account was not unlocked').to.be.true;
+
+    const latestActiveForkName = getLatestActiveForkName();
+    const bridge = getBridge(rskTxHelper.getClient(), latestActiveForkName);
+    const federationAddress = await bridge.methods.getFederationAddress().call();
+
+    const federationAddressBalanceInitial = Number(await btcTxHelper.getAddressBalance(federationAddress));
+
+    await btcTxHelper.fundAddress(btcAddressInfo.address, PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
+    
+    const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, btcAddressInfo, PEGIN_VALUE_IN_BTC);
+    await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+    const recipientRskAddressBalanceAfterPegin = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));
+    expect(Number(recipientRskAddressBalanceAfterPegin)).to.be.equal(Number(3 * MINIMUM_PEGOUT_VALUE_IN_WEIS));
+
+    await sendTxToBridge(rskTxHelper, PEGOUT_VALUE_IN_RBTC, recipientRskAddressInfo.address);
+    await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
+
+    const federationAddressBalanceAfterFirstPegout = Number(await btcTxHelper.getAddressBalance(federationAddress));
+    expect(Number(federationAddressBalanceAfterFirstPegout)).to.be.equal(Number(federationAddressBalanceInitial + 2 * MINIMUM_PEGIN_VALUE_IN_BTC));
+
+    const senderAddressBalanceAfterFirstPegout = Number(await btcTxHelper.getAddressBalance(btcAddressInfo.address));
+    expect(Number(senderAddressBalanceAfterFirstPegout)).to.be.above(MINIMUM_PEGIN_VALUE_IN_BTC - btcTxHelper.getFee()).and.below(MINIMUM_PEGIN_VALUE_IN_BTC);
+
+    const recipientRskAddressAfterFirstPegout = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));
+    expect(Number(recipientRskAddressAfterFirstPegout)).to.be.above(MINIMUM_PEGOUT_VALUE_IN_WEIS - RSK_TX_FEE_IN_RBTC).and.below(2 * MINIMUM_PEGOUT_VALUE_IN_WEIS);
+    
+    await sendTxToBridge(rskTxHelper, PEGOUT_VALUE_IN_RBTC, recipientRskAddressInfo.address);
+    await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
+
+    const senderAddressBalanceAfterSecondPegout = Number(await btcTxHelper.getAddressBalance(btcAddressInfo.address));
+    expect(Number(senderAddressBalanceAfterSecondPegout)).to.be.above(2*MINIMUM_PEGIN_VALUE_IN_BTC - 2 * btcTxHelper.getFee()).and.below(2 * MINIMUM_PEGIN_VALUE_IN_BTC);
```

Comment:
This assertion is difficult to follow due to the math operations. Try creating variables for those expected values and exploiting the two conditions into two separate assertions; one checking is above the expected value, and another checking is below the expected value.